core/src/test/java/com/spotify/ffwd/serializer/TestSpotify100ProtoSerializer.java

```diff
@@ -176,57 +144,13 @@ public void testSerializedMetricWithDistributionValue() throws Exception {
   }
 
 
-  @Test
```

Comment:
should the removed tests in here be replaced or are they covered by something else now?